shell/browser/electron_browser_client.cc

```diff
@@ -524,6 +524,11 @@ void ElectronBrowserClient::OverrideWebkitPrefs(
   prefs->picture_in_picture_enabled = false;
 #endif
 
+  ui::NativeTheme* native_theme = ui::NativeTheme::GetInstanceForNativeUi();
+  prefs->preferred_color_scheme = native_theme->ShouldUseDarkColors()
```

Comment:
Can you use the new pipeline to set the color scheme for renderer process as mentioned in https://chromium-review.googlesource.com/c/chromium/src/+/1845575